.travis/parse_tag.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+set -euo pipefail
+
+readonly TRAVIS_TAG=${TRAVIS_TAG:-}
+
+if [[ -z "$TRAVIS_TAG" ]]; then
+    echo "TRAVIS_TAG is not defined, abort."
+else
+    regex="^deploy_server_(.*)_env_(.*)$"
+    
+    if [[ $TRAVIS_TAG =~ $regex ]]; then
+        export SWAP_SERVER="${BASH_REMATCH[1]}"
+        export SWAP_ENV="${BASH_REMATCH[2]}"
+        echo server: $SWAP_SERVER env: $SWAP_ENV
```

Comment:
when there is formatting directive, using printf is always better.

Example: `printf 'server: %s env: %s' "$SWAP_SERVER" "$SWAP_ENV"`.

Reply:
I've modified  this printing.